test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
I made this change as a precaution that out tests didn't use any non-public headers accidentally. Instead forcing their inclusion to be somewhat more explicit.

I'll leave it up to you which you would rather have.